recipes/treematching/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "treematching" %}
+{% set version = "0.1.20210513" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/LionelAuroux/treematching
```

Comment:
Can you build this from a Github tarball instead? That might help with the Windows issue.

Reply:
There isn't a tarball for this on project on github, there only seems to be a master tag https://github.com/LionelAuroux/treematching

Reply:
There is always one for a commit automatically, e.g. https://github.com/LionelAuroux/treematching/archive/5ebc2474d8720d87569cf414bd0275010edf8f19.tar.gz